src/program/programs/store/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use anchor_lang::prelude::*;
+
+declare_id!("Ai6qd7Z2TYALRkfG5NTyNd5hoUK2wUCBtJd8d5wZAvpJ");
+
+#[program]
+pub mod store {
+    use super::*;
+    // handler function
+    pub fn create_customer(
+        ctx: Context<CreateUserStats>,
+        name: String,
+        item: String,
+    ) -> Result<()> {
+        let customer = &mut ctx.accounts.customer;
+        if name.as_bytes().len() > 200 {
+            panic!();
```

Comment:
Would be better if you throw an Error here instead of panic

Reply:
okay, will update that